src/event.jl

```diff
@@ -35,9 +35,11 @@ struct LogEvent
         end
 
         if sizeof(message) > MAX_EVENT_SIZE - 26
-            throw(ArgumentError(
-                "Log Event message cannot be more than than $MAX_EVENT_SIZE bytes"
-            ))
+            # Truncate message and throw a warning
+            idx = MAX_EVENT_SIZE - 29
+            # Truncated messages include a "..."
+            message = string(message[1:idx], "...")
+            warn(LOGGER, "Log Event message cannot be more than $MAX_EVENT_SIZE bytes")
```

Comment:
```suggestion
            warn(LOGGER, "CloudWatch Log Event message cannot be more than $MAX_EVENT_SIZE bytes")
```
Sorry, I squashed this into my previous suggestion but didn't comment on it specifically